students/JeroenSweerts/Lesson3/List_Lab_Exercise.py

```diff
@@ -1,11 +1,110 @@
 #!/usr/bin/env python3
-
+##########################SERIES1###############################################
+'''Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.'''
 fruitlist = ['Apples', 'Pears', 'Oranges','Peaches']
 
+'''Display the list (plain old print() is fine…).'''
 print(fruitlist)
+
+'''Ask the user for another fruit and add it to the end of the list.'''
 response = input("Please enter a fruit to add to the list > ")
 fruitlist.append(response)
+
+'''Display the list.'''
+print("New list with appended fruit > "+response)
 print(fruitlist)
+print('')
 
+'''Ask the user for a number and display the number back to the user and
+    the fruit corresponding to that number (on a 1-is-first basis).
+    Remember that Python uses zero-based indexing, so you will need to correct.'''
 response = input("Please enter the number of the fruit of the list to display > ")
-print("fruit " + response + " : " + fruitlist[int(response)-1])
+print("you selected fruit " + response + " : " + fruitlist[int(response)-1])
+
+'''Add another fruit to the beginning of the list using “+”
+and display the list.'''
+fruitlist = ['banana'] + fruitlist
+print("New list with banana added as first element: ")
+print(fruitlist)
+print('')
+
+'''Add another fruit to the beginning of the list using insert()
+and display the list.'''
+fruitlist.insert(0, 'Cherries')
+print("New list with Cherries added as first element: ")
+print(fruitlist)
+print('')
+
+'''Display all the fruits that begin with “P”, using a for loop.'''
+print("Now we display all fruits starting with a P")
+for fruits in fruitlist:
+    if fruits[0] == 'P' or fruits[0] == 'p':
```

Comment:
you could also do 

`if fruits[0].lower() == 'p'` or `if fruits[0].lower().startswith('p')`